docs/source/models/field_ionization.rst

```diff
@@ -56,6 +56,23 @@ Overview: Implemented Models
 
     Models marked with "(R&D)" are under *research and development* and should be used with care.
 
+Ionization Current
+------------------
+
+In order to conserve energy, PIConGPU allows to activate an ionization current to weaken the electric field for every field ionization model.
+The current for a single ion is
+
+.. math::
+
+    \mathbf{J}_\mathrm{ion} = E_\mathrm{ion} \frac{\mathbf{E}}{\Delta t |\mathbf{E}|^2}
```

Comment:
Add division by cell volume and explain in the text below that we assume that the ejection energy is zero.